sound/soc/sof/ipc3-dtrace.c

```diff
@@ -224,6 +224,21 @@ static int debugfs_create_trace_filter(struct snd_sof_dev *sdev)
 	return 0;
 }
 
+static bool sof_dtrace_set_host_offset(struct snd_sof_dev *sdev, u32 new_offset)
+{
+	u32 host_offset = READ_ONCE(sdev->host_offset);
```

Comment:
so this is going to conflict with your very own PR #3662 changes, specifically https://github.com/thesofproject/linux/pull/3632/commits/024c2eb3f8674e791dabf05c73f3ee58554f6238

Reply:
Yes, I'm aware of that, they are of different topic and I don't want to mix them. I'm fine with fixing up either when the time comes.